README.md

```diff
@@ -31,6 +31,8 @@ For more information please refer to the main [Apiary](https://github.com/Expedi
 | k8s_max_replica_count                        | Max Number of k8s pod replicas to create.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | number |                  `10`                  |    no    |
+| k8s_svc_spec                                 | Waggledance Kubernetes service settings. All inner fields are optional and if unset the kubernetes default values are applied.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | object |                  `-`                  |    no    |
+| k8s_svc_annotations                          | Custom annotations for the Waggledance Kubernetes service..                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | map(string) |                  `"service.beta.kubernetes.io/aws-load-balancer-type"     = "nlb"`                  |    no    |
```

Comment:
```suggestion
| k8s_svc_annotations                          | Custom annotations for the Waggledance Kubernetes service.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | map(string) |                  `"service.beta.kubernetes.io/aws-load-balancer-type"     = "nlb"`                  |    no    |
```

Reply:
what about the: https://github.com/ExpediaGroup/apiary-federation/pull/147/files#diff-05b5a57c136b6ff596500bcbfdcff145ef6cddea2a0e86d184d9daa9a65a288eR137